### PR TITLE
Add env variable RUN_URL, TEST_RESULTS_DIR and TEST_RESULTS_FILE

### PR DESCRIPTION
**Pull request by anson627**

### CODE REVIEW

**Review by sumanthreddy29 — DISMISSED** [review]

**Review by alyssa1303 — COMMENTED** [review]
Can you make a change to update the order of RUN_ID to this instead `run_id=$(Build.BuildId)-$(System.JobId)`

**Comment by anson627** [comment]
> Can you make a change to update the order of RUN_ID to this instead `run_id=$(Build.BuildId)-$(System.JobId)`

Done

File: steps/cloud/azure/upload-storage-account.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ steps:
     fi
   displayName: "Upload Result"
   env:
-    SOURCE_FILE_NAME: ${{ parameters.source_file_name }}
+    SOURCE_FILE_NAME: $(TEST_RESULTS_FILE)
```
Comment (sumanthreddy29) [inline steps/cloud/azure/upload-storage-account.yml:31]:
```suggestion
    SOURCE_FILE_NAME: ${{ parameters.source_file_name }}
```
can we revert this change, so that we allow user to override the file name ?

Comment (sumanthreddy29) [reply steps/cloud/azure/upload-storage-account.yml:31]: If we don't want user to override, we might have to remove parameter and use the env variable `TEST_RESULTS_FILE` in all the places

Comment (anson627) [reply steps/cloud/azure/upload-storage-account.yml]: reverted, nice catch!

File: steps/engine/kperf/execute.yml [hunk 2]
```diff
@@ -79,7 +77,6 @@ steps:
     BENCHMARK_SUBCMD: ${{ parameters.engine_input.benchmark_subcmd }}
     BENCHMARK_SUBCMD_ARGS: ${{ parameters.engine_input.benchmark_subcmd_args }}
     EXTRA_BENCHMARK_SUBCMD_ARGS: ${{ parameters.extra_benchmark_subcmd_args }}
-    RESULT_DIR: $(System.DefaultWorkingDirectory)/$(RUN_ID)
+    RESULT_DIR: $(TEST_RESULTS_DIR)
```
Comment (alyssa1303) [inline steps/engine/kperf/execute.yml:80]: Maybe you can use TEST_RESULTS_DIR directly in the script like the collect step and get rid of this env var?

**Review by sumanthreddy29 — APPROVED** [review]
